<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SdrDeviceLimesdrRxctl</class>
 <widget class="QWidget" name="SdrDeviceLimesdrRxctl">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>207</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="gainLabel">
       <property name="text">
        <string>Gain</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="gainValueLabel">
       <property name="toolTip">
        <string>Receiver gain value</string>
       </property>
       <property name="text">
        <string>0 dB</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QSlider" name="gainSlider">
       <property name="toolTip">
        <string>Adjust receiver gain</string>
       </property>
       <property name="maximum">
        <number>73</number>
       </property>
       <property name="pageStep">
        <number>5</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksBelow</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>10</number>
     </property>
     <item>
      <widget class="QCheckBox" name="lpfCheckBox">
       <property name="toolTip">
        <string>Enable or disable the analog low-pass filter in the LMS7002D chip</string>
       </property>
       <property name="text">
        <string>Analog LPF</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="gfirCheckBox">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="toolTip">
        <string>Enable or disable the digital low-pass filter in the LMS7002D chip</string>
       </property>
       <property name="text">
        <string>Digital LPF</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="calButton">
     <property name="toolTip">
      <string>Run automatic calibration of receiver path</string>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
